src/sailor/db/elastic_model.lua

```diff
@@ -0,0 +1,50 @@
+local elastic = require "sailor.db.elasticsearch"
+local Emodel = {}
```

Comment:
if this is the elasticsearch model, we usually use `ES` instead of E. Also when you have big name variables you usually use lower case with underscore. So maybe `es_model` would be a better naming here.